connector/http-agent/app/express/interceptors/deviceIdentification.js

```diff
@@ -14,7 +14,8 @@ const tls = require('tls');
 module.exports = ({ cache, config }) => ({
   name: 'device-identification-interceptor',
   middleware: async (req, res, next) => {
-    const err = new createError.Unauthorized();
+    const Forbidden = new createError[403]();
+    const BadRequest = new createError[400]();
```

Comment:
It's creating 2 new Error Messages in all requests. It could be placed at the beginning of the file, as 'global' or...